mod_reforged/config/item_names.nut

```diff
@@ -1,3 +1,52 @@
+// Vanilla Adjustments
+::MSU.Array.removeValues(::Const.Strings.CrossbowNames, [
+	"Betrayer",		// moved over to dagger names
+]);
+::Const.Strings.CrossbowNames.extend([
+	"Scorpion",
+	"Ironsight",
+	"Windlass",
+]);
+
+::MSU.Array.removeValues(::Const.Strings.DaggerNames, [
+	"Estoc",		// We have our own actual Kriegsmesser item
+]);
+::Const.Strings.DaggerNames.extend([
+	"Betrayer",		// formerly a crossbow name
+]);
+
+::Const.Strings.HandgonneNames.extend([
+	"Deafener",
+	"Eisenknall",
+	"Volcano",
+]);
+
+::Const.Strings.LongaxeNames.extend([
+	"Culler",
+]);
+
+::Const.Strings.SwordlanceNames.extend([	// In Vanilla this has only 2 entries
+	"Bloodmoon",
+	"Ghoul\'s Bane",
```

Comment:
I don't think we can use 2 word names because in vanilla it can do things like `Sir Roderick's xyz` and there `Sir Roderick's Ghoul's Bane` sounds weird. I'd stick to single word names.

Reply:
Vanilla uses 2 word names plenty of times, sometimes even 3 word names.

And for greatswords they even have `"Thug\'s Bane",`
![image](https://github.com/Battle-Modders/mod-reforged/assets/2252464/3b21deaa-bd21-4797-ae80-4788cd5f4579)

Reply:
I see. In that case I'd not use the word "Ghoul" as it seems out of place for southern faction lore in BB. Maybe something else's bane e.g.:
- Nomad's Bane
- Mortal's Bane

Or remove the name if you don't find any other appropriate replacement for Ghoul.

Reply:
I looked through the vanilla codebase and found that the word `ghoul` is used in the text of the `roaming_beasts_desert_contract` and the word `ghoulish` is used in many other texts. So, `Ghoul\'s Bane` is fine.